README.md

```diff
@@ -12,19 +12,98 @@ other components of the system, including the Istio mixer and the Istio proxy me
 If you have a question about the Istio Manager or have a problem using it, please
 [file an issue](https://github.com/istio/manager/issues/new).
 
-## Build instructions ##
+## Setting up a local dev environment ##
+
+_Note:_ This section applies to Mac and Windows users only.
+
+### Pre-requisites ##
+
+- Setup Go 1.7.5+ on your host machine
+- Clone this repository
+- Install [Virtualbox](https://github.com/kubernetes/minikube/releases)
+- Install [Minikube](https://github.com/kubernetes/minikube/releases)
+- Install [Vagrant](https://www.vagrantup.com/downloads.html)
+- Install [kubectl](https://kubernetes.io/docs/user-guide/prereqs/)
+
+### 1. Start Minikube
+
+    minikube config set kubernetes-version v1.5.2
+    minikube start
+    
+Copy the kube config file to the platform/kube directory and update the paths
+
+    cp ~/.kube/config platform/kube/
+    sed -i 's!/Users/<username>!/home/ubuntu!' platform/kube/config
+
+_Note_: The `sed` command above may not work on Windows machines. Replace
+the path to certs such that the resultant paths look like
+`/home/ubuntu/.minikube/ca.crt`, etc.
```

Comment:
Maybe we should just symlink the whole directory? But then paths are hard-coded in .kube/config...